tests/installation/bootloader_svirt.pm

```diff
@@ -56,6 +57,31 @@ sub search_image_on_svirt_host {
     return $path;
 }
 
+# wrapper for vmware shell cmd execution
+# res: 0 passed, 1 failed. Does 'assertion' when errmsg provided
+sub check_run_command {
+    my ($self, $command, $err_msg) = @_;
+    my $res = $self->run_cmd($command, domain => 'sshVMwareServer');
+    croak($err_msg) if ($res && $err_msg);
```

Comment:
I find this highly confusing. Passing an error message alters the behaviour of the function from "return value" to "die" appears a bit out-of-the ordinary.

For the sake of readability I would suggest to create two helper functions, `run_command` and `assert_run_command` because this naming should avoid confusion afterwards.

IMHO is passing an error message and by doing so making a run function `die` a unintended side-effect that I find confusing.

Reply:
Added parameter `assert => 1` to let the call act as an assertion.